packages/globals/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from './globals'
+export { KeyboardLayout, KeyCode, US_INTERNATIONAL } from './keyboard-layout'
```

Comment:
Maybe it's better to move it in a separate package `@interactors/keyboard`?

Reply:
I think that this is core to `html` as this keyboard interactor is very much tied to the DOM. We could move it to another package, but `fillIn` currently needs it, so we'd have a circular dependency without a lot of work.